### PR TITLE
Issue #999 c128 applied to c129

### PR DESCRIPTION
**Pull request by franzhollerer**

### CODE REVIEW

File: CppCoreGuidelines.md [hunk 1]
```diff
@@ -6789,7 +6789,7 @@ First we devise a hierarchy of interface classes:
 
     class Circle : public Shape {   // pure interface
     public:
-        int radius() = 0;
+        virutal int radius() = 0;
```
Comment (dalg24) [inline CppCoreGuidelines.md]: `virtual`

Comment (franzhollerer) [reply CppCoreGuidelines.md]: thanks. fixed

**Comment by gdr-at-ms** [comment]
Looks good, could you squash it?

**Comment by jwakely** [comment]
I did a "squash and merge" to squash the commits into one.

(The alternative would have been for Franz to rebase his branch and force push it as a single commit.)

**Comment by franzhollerer** [comment]
Many thanks at @jwakely. I had put it on my TODO list. But as I am not that familar with git I am glad that you addressed it in the meantime.

**Comment by gdr-at-ms** [comment]
Thanks!

**Pull request merged**